### ISSUE TITLE
Speaking not detected (Android Emulator)

### ISSUE THREAD

**Issue by BurakAkten** [csdcorp/speech_to_text#197]
First of all , the code is the same one from example part.  After click the start button, the bottom label changing to "I'm listening" but no word were detected. 

The console outputs: 
`
D/SpeechToTextPlugin(12038): Start listening
D/SpeechToTextPlugin(12038): setupRecognizerIntent
D/SpeechToTextPlugin(12038): Notify listening
D/SpeechToTextPlugin(12038): Notify listening done
D/SpeechToTextPlugin(12038): Start listening done
D/SpeechToTextPlugin(12038): In RecognizerIntent apply
D/SpeechToTextPlugin(12038): put model
D/SpeechToTextPlugin(12038): put package
D/SpeechToTextPlugin(12038): put partial
D/SpeechToTextPlugin(12038): In RecognizerIntent apply
D/SpeechToTextPlugin(12038): put model
D/SpeechToTextPlugin(12038): put package
D/SpeechToTextPlugin(12038): put partial
D/SpeechToTextPlugin(12038): put languageTag
D/SpeechToTextPlugin(12038): rmsDB -2.0 / -2.0
I/chatty  (12038): uid=10135(com.bakten.flutter_app_speech) identical 1 line
D/SpeechToTextPlugin(12038): rmsDB -2.0 / -2.0
D/SpeechToTextPlugin(12038): rmsDB -2.12 / -2.0
I/chatty  (12038): uid=10135(com.example.flutter_app_speech) identical 5 lines
D/SpeechToTextPlugin(12038): rmsDB -2.12 / -2.0
I/chatty  (12038): uid=10135(com.example.flutter_app_speech) identical 17 lines
D/SpeechToTextPlugin(12038): rmsDB -2.12 / -2.0
I/chatty  (12038): uid=10135(com.example.flutter_app_speech) identical 17 lines
D/SpeechToTextPlugin(12038): rmsDB -2.12 / -2.0
I/chatty  (12038): uid=10135(com.example.flutter_app_speech) identical 24 lines
D/SpeechToTextPlugin(12038): rmsDB -2.12 / -2.0
D/SpeechToTextPlugin(12038): Stop listening
D/SpeechToTextPlugin(12038): Notify listening
D/SpeechToTextPlugin(12038): Notify listening done
D/SpeechToTextPlugin(12038): Stop listening done
D/SpeechToTextPlugin(12038): Recognizer destroy
`

Flutter 1.22.6 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 9b2d32b605 (9 weeks ago) • 2021-01-22 14:36:39 -0800
Engine • revision 2f0af37152
Tools • Dart 2.10.5

**Comment by sowens-csd** [comment]
Have you been through the troubleshooting section of the README for tips on setting up the speech recognizer on Android emulators?

**Comment by BurakAkten** [comment]
Yes I follow thats suggesttion. Then the logs gone but different diffucilties appeared. Then I tried to run on iOS simulator then I had this errors "Incorrect Swift version trying to compile for iOS" that mentioned on ReadMe. After that I followed this thread -> https://github.com/csdcorp/speech_to_text/issues/45 and I used 1.0.1 version of speech_to_text. This mean I can't use some of feature. Now I can make speech to text but I can't use some necessary properties. Is there any solution for it for newer version of v.1.0.1.

Thank you for your quick response.
